examples/TextClassification/README.md

```diff
@@ -1,56 +1,41 @@
 # Deploying Text Classification model
```

Comment:
Please confirm steps have not been lost from previous version

Reply:
Have checked all the readme files. I have run the commands also, it was working.